reference/conanfile/methods.rst

```diff
@@ -1508,6 +1508,14 @@ self.folders
   method in a ``conanfile.py``. In the cache, when running the :command:`conan create`, this subfolder will be relative to the root
   build folder and when running the :command:`conan imports` command it will be relative to the current working directory.
 
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
+
+- **self.folders.root** (Defaulted to None): Specifies a parent directory where the sources, generators, etc are located specifically when the ``conanfile.py`` is located in a spereated subdirectory.
+
+Available since: `1.51.0 <https://github.com/conan-io/conan/releases>`_
+
+- **self.folders.subproject** (Defaulted to None): Specifies a subfolder where to the ``conanfile.py`` relative  to the project root. This is particulary useful for :ref:`package_layout<example_multiple_subprojects>`.
```

Comment:
```suggestion
- **self.folders.subproject** (Defaulted to None): Specifies a subfolder where the ``conanfile.py`` is relative to the project root. This is particularly useful for :ref:`package_layout<example_multiple_subprojects>`.
```

Reply:
Apart from that, I would be more verbose with that reference, for instance, `package_layout_example_multiple_subprojects`.

Then, remember that you'll have to add that new one in `developing_packages/package_layout.rst` file before the `Example: Multiple subprojects` section:

```
              folder = os.path.join(self.recipe_folder, "..")
              copy(self, "*.txt", folder, self.export_sources_folder)
          
          def source(self):
              # we can see that the CMakeLists.txt is inside the source folder
              cmake = load(self, "CMakeLists.txt")
          def build(self):
              # The build() method can also access the CMakeLists.txt in the source folder
              path = os.path.join(self.source_folder, "CMakeLists.txt")
              cmake = load(self, path)



.. _package_layout_example_multiple_subprojects:

Example: Multiple subprojects
-----------------------------

Lets say that we have a project that contains multiple subprojects, and some of these subprojects need
to access some information that is at their same level (sibling folders). Each subproject would be 
a Conan package.
```

Finally, that part should include this new reference. For instance:

```
useful for :ref:`layouts with multiple subprojects<package_layout_example_multiple_subprojects>`
```

Reply:
Let me take a stab at this, the directions are really good starting point for a first time